doc/manuals/admin/build_source.md

```diff
@@ -53,6 +53,7 @@ commands that require root privilege):
 
 * Install Google Test/Mock from sources (there are RPM packages for this, but they do not work with the current CMakeLists.txt configuration). Previously the URL was http://googlemock.googlecode.com/files/gmock-1.5.0.tar.bz2 but Google removed that package in late August 2016 and it is no longer working.
 
+        yum install perl-Digest-MD5 libxslt
```

Comment:
I understand this refers to the standard procedure to build Orion from soures (i.e. not specific for aarch64).b Could you elaborate on the need of these packages, please?

Reply:
In the case of aarch64, as the following errors occurred in coverage, they are needed.
- perl-Digest-MD5
```
make[1]: Leaving directory `/opt/fiware-orion/BUILD_COVERAGE'
# Init coverage
echo "Initializing coverage files"
Initializing coverage files
mkdir -p coverage
lcov -i --zerocounters --directory BUILD_COVERAGE/
Deleting all .da files in BUILD_COVERAGE/ and subdirectories
Done.
lcov --capture --initial --directory BUILD_COVERAGE -b BUILD_COVERAGE --output-file coverage/broker.init.info
Capturing coverage data from BUILD_COVERAGE
Can't locate Digest/MD5.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at /usr/bin/geninfo line 57.
BEGIN failed--compilation aborted at /usr/bin/geninfo line 57. 
make: *** [coverage] Error 2
```
- libxslt
```
        1119 - 3389_forced_update_option_bool (Failed)
        1120 - 3389_forced_update_option_number (Failed)
        1121 - 3389_forced_update_option_string (Failed)
        1132 - legacy-forwarding-with-idpattern (Failed)
Errors while running CTest
make[1]: *** [test] Error 8
make[1]: Leaving directory `/opt/fiware-orion/BUILD_COVERAGE'
/bin/sh: xsltproc: command not found
make: *** [coverage] Error 127 
```

Reply:
Ok. NTC